client/components/conditions/add-conditions.js

```diff
@@ -118,9 +121,10 @@ function AddConditions({
           onAdd={handleOnAdd}
           onUpdate={handleOnUpdate}
           onRemove={handleOnRemove}
+          conditionKey={'uuid'}
```

Comment:
What does this do?

Reply:
This gives the custom condition a temporary condition key so a reminder can be saved against it in the state, then in the save function it is replaced when the UUID is actually created. Initially I tried to give it a real UUID at this point but it generated a new one every time it was rendered which broke things, so I opted for this way.

Reply:
I think I see. The eventual uuid is generated at L142, but the downstream logic in the editable component wants a `conditionKey` to be set to allow reminders to be added. Is that right?

If so, could you just add a comment to the effect that the string being passed is an arbitrary placeholder and there's nothing special about the value `'uuid'`?

Reply:
Correct. Have added a comment above the line